packages/react-components/react-skeleton/stories/Skeleton/SkeletonRow.stories.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import { Skeleton, SkeletonItem, SkeletonProps } from '@fluentui/react-skeleton';
+import { makeStyles, shorthands, tokens } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+  invertedWrapper: {
+    backgroundColor: tokens.colorNeutralBackground1,
+  },
+  firstRow: {
+    alignItems: 'center',
+    display: 'grid',
+    paddingBottom: '10px',
+    position: 'relative',
+    ...shorthands.gap('5px'),
+    ...shorthands.margin('5px'),
+    gridTemplateColumns: 'min-content 80%',
+  },
+  secondRow: {
+    alignItems: 'center',
+    display: 'grid',
+    paddingBottom: '10px',
+    position: 'relative',
+    ...shorthands.gap('5px'),
+    ...shorthands.margin('5px'),
+    gridTemplateColumns: 'min-content 20% 20% 15% 15%',
+  },
+});
+
+const SkeletonElementsFirstRow = () => {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.firstRow}>
+      <SkeletonItem shape="circle" size={24} />
+      <SkeletonItem shape="rectangle" size={16} />
+    </div>
+  );
+};
+
+const SkeletonElementsSecondRow = () => {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.secondRow}>
+      <SkeletonItem shape="circle" size={24} />
+      <SkeletonItem size={16} />
+      <SkeletonItem size={16} />
+      <SkeletonItem size={16} />
+      <SkeletonItem size={16} />
+    </div>
+  );
+};
+
+const SkeletonElementsThirdRow = () => {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.secondRow}>
```

Comment:
Is this correct that the third row is using a class called `secondRow`? Maybe rename the class to be `secondThirdRow` or something so it's clearer?